modules/sr/robot/motor_devices.py

```diff
@@ -36,6 +46,13 @@ def set_speed(self, speed):
 
 
 class Gripper(MotorBase):
+    """
+        Our default robot's gripper has its fingers attached to linear motors.
+        This class allows you to control these as a single entity to open/close.
+
+        You should initialise with the motor_name as the two Webots motor names separated by
+        a pipe character (|)
```

Comment:
Hrm, I'd not spotted this. We should perhaps change this to being a 2-tuple rather than requiring string parsing? Not for this PR though.

Reply:
#198 fixes this.